source/Nuke.Common/ChangeLog/ChangeLog.cs

```diff
@@ -25,6 +25,10 @@ public ChangeLog(string path, IReadOnlyList<ReleaseNotes> releaseNotes)
 
         public string Path { get; }
         [CanBeNull] public ReleaseNotes Unreleased { get; }
+        
+        /// <summary>
+        /// Release notes with defined version and sorted by it.
+        /// </summary>
```

Comment:
I don't really like the last part of the summary. What do you think about `Release notes sorted by version.`.